app/views/admin/conferences/edit.html.haml

```diff
@@ -24,5 +24,5 @@
         = f.input :start_hour, input_html: {size: 2, type: 'number', min: 0, max: 23}
         = f.input :end_hour, input_html: {size: 2, type: 'number', min: 1, max: 24}
       = f.inputs name: 'Registrations' do
-        = f.input :registration_limit, as: :number, in: 0..9999, hint: 'Limit the number of registrations to the conference (0 no limit). You currently have ' + pluralize(@conference.registrations.count, 'registration')
+        = f.input :registration_limit, as: :number, in: 0..9999, hint: 'Limit the number of registrations to the conference (0 no limit). Please note that the registration limit doesn\'t apply to speakers of confirmed events. You currently have ' + pluralize(@conference.registrations.count, 'registration')
```

Comment:
Let's clarify what we mean:

Please note that the registration limit doesn\'t apply to speakers of confirmed events **, and they will still be able to register.**